libs/langchain/langchain/retrievers/google_vertex_ai_search.py

```diff
@@ -273,3 +305,20 @@ def _get_relevant_documents(
             )
 
         return documents
+
+
+class GoogleCloudEnterpriseSearchRetriever(GoogleVertexAISearchRetriever):
```

Comment:
can we keep this in old file for backwards compatibility

Reply:
Sorry, missed this, I'll update it in another commit.

Reply:
not seeing it, maybe im missing something?

Reply:
I added it, then I removed it because it was confusing early users.

Reply:
this is a breaking change if we don't include that, would prefer to avoid breaks if easy